### DISCUSSION TITLE
Huggingface gradiyo error

### DISCUSSION THREAD

**Discussion by Dipak1881993** [community/community#42070] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

![Screenshot_2022-12-18-23-20-40-90_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/117624758/208312027-6f0fcac3-8786-43f7-b202-ac284ed7ef30.jpg)

**Comment by Akash1134** [comment]
Hi @Dipak1881993  ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
